.github/workflows/update-js-deps.yml

```diff
@@ -0,0 +1,139 @@
+name: js-upgrade
+
+on:
+  workflow_call:
```

Comment:
Shouldn't this be on a schedule?

Reply:
Schedule is set on the module - [see](https://github.com/silverstripe/github-actions-ci-cd/pull/3/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R64)